pages/android/bugreport.md

```diff
@@ -0,0 +1,8 @@
+# bugreport
+
+> Show an Android bug report through the adb shell.
```

Comment:
The `adb shell` is not required to use this command.

Reply:
But it isn't installed natively. Where is the command comming from then?

Reply:
The `adb shell` don't install software on the target device as far i know.

Reply:
I have no idea how adb works, but I would have guessed, that `bugreport` runs only in the `adb shell` and gets the information from the device with some lower level commands.

I was just worried that it might seem, that it's an native android command (Maybe it even is, but I can't execute it in Termux)

Reply:
Hrm. If you can't execute it in a local native shell in Termux, then I'm guessing that it should go under `adb-shell-bugreport`? But that makes it inconsistent, and then it calls the other Android pages into question....? I'm confused on this one.

Reply:
I checked our other Android pages and they are all natively installed. Personally I think it makes more sense putting `adb shell` commands in the Android folder, but explicitly mention that the command is only available for `adb shell`. Should we maybe open in issue for this?

Reply:
Yeah, that sounds like an acceptable compromise @marchersimon. Yeah, I'd suggest opening an issue for transparency about this problem.